pkg/resources/csi/kubevirt/deployment.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	csiVersion = "cc71b72b8d5a205685985244c61707c5e40c9d5f"
+	csiVersion = "latest"
```

Comment:
Please find a different tag than this, `latest` will cause endless problems down the road for us.

Reply:
Pushed the image to kubermatic/kubevirt-csi-driver repo and add a version tag in this [commit](https://github.com/kubermatic/kubermatic/pull/13818/commits/9478e71e59ad54a2c6ba4e64681affa7cba3cf7b)